source/postcard-rpc/src/host_client/raw_nusb.rs

```diff
@@ -111,11 +111,31 @@ where
             .claim_interface(interface_id as u8)
             .map_err(|e| format!("Failed claiming interface: {e:?}"))?;
 
+        let mut mps: Option<usize> = None;
+        if let Ok(config) = dev.active_configuration() {
+            for ias in config.interface_alt_settings() {
+                for ep in ias.endpoints() {
+                    if ep.address() == BULK_OUT_EP {
+                        mps = Some(match mps.take() {
+                            Some(old) => old.min(ep.max_packet_size()),
+                            None => ep.max_packet_size(),
+                        });
+                    }
+                }
+            }
+        }
+
+        let Some(max_packet_size) = mps else {
```

Comment:
I could probably actually store a `None` to the out worker, and make it emit a ZLP on every send, if we can't determine max packet size. That would be less performant, but probably better than just rejecting the connection.